docs/cni/kubernetes/vagrant-coreos/VagrantCoreOS.md

```diff
@@ -8,110 +8,9 @@
 
```

Comment:
I didn't remove / modify any images. This was broken before I go to it. Let's keep it out of scope for this PR.